app/src/test/java/ch/epfl/qedit/model/UserTest.java

```diff
@@ -27,7 +27,7 @@ public void dummyTestUser() {
     }
 
     @Test
-    public void testConstructor() {
+    public void testConstructor1() {
```

Comment:
There is only one of them but who cares ;)

Reply:
oh yes right XD